akka-actor/src/main/scala/akka/actor/Deployer.scala

```diff
@@ -109,6 +110,16 @@ case object LocalScope extends LocalScope {
   def withFallback(other: Scope): Scope = this
 }
 
+/**
+ * INTERNAL API
+ *
+ * Used for disallowing remote deployment for adapted typed actors
+ */
+@InternalApi
+private[akka] case object DisallowRemoteScope extends LocalScope {
```

Comment:
this isn't used (yet)?

Reply:
Argh, tired Friday PR, was from my previous attempt to solve it.